### PR TITLE
feat(detail and html screens): condition for landscape mode

### PR DESCRIPTION
**Pull request by SoleCincis**
- created a ( possibly temporary ) style component to wrap around detail and html screens if on landscape mode
  - WrapperLandscape is increasing space left and right in order to center the text to improve rideability
- it's applied on Detail and HTML screens with necessary condition
  - if device is on landscape then content of this screens is wrapped

SVA-93

- on OrientationProvider just a small grammar correction on comment
- WrapperLandscape it's updated with percentage to fit bigger screens as well
- WrapperWithOrientation is a function component that carries WrapperLandscape and pass it to its children
   - this happens when orientation is landscape , which reach its children per prop
- wrapping detail and html screens with new component WrapperWithOrientation instead of WrapperLandscape
  - wrapping will be effective just when orientation prop is triggered , in this way we are avoid code repetition
- WrapperWithOrientation is implemented in formScreen es well

SVA-93

fix: position of OrientationContext

- moved useContext on top of the file in order to prevent future bugs and errors

SVA-93

feat(html view): size images in html views

- added calculation for image width depending on orientation
  in html views

SVA-93

feat(images carousel): center carousel on landscape

- centered the carousel in every case
- problem was with safe areas, where the device width does
  not equal the available area width

SVA-93

feat(detail screens): add landscape wrapper for everything except ImagesCarousel

- removed wrapper from `DetailScreen` around `Component` because
  it was also wrapping the inner `ImagesCarousel`
- added landscape wrapper in every detail component

SVA-93

### CODE REVIEW

**Comment by SoleCincis** [comment]
HTML screen ( iPhone 11 Pro Max )
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-05 at 18 50 59

**Comment by SoleCincis** [comment]
Detail screen ( iPhone 11 Pro Max ) 
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 52 08
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 52 20

**Comment by SoleCincis** [comment]
HTML screen with Image ( iPhone 11 Pro Max ) @donni106 do you think Image should be smaller ? 
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 54 28
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 53 41

File: src/components/Wrapper.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ export const WrapperHorizontal = styled.View`
   padding-right: ${normalize(14)}px;
 `;
 
+export const WrapperLandscape = styled.View`
+  padding-left: ${normalize(150)}px;
+  padding-right: ${normalize(150)}px;
```
Comment (donni106) [inline src/components/Wrapper.js:16]: let's use percentages here, to have the same ratios on different screen sizes.

**Comment by donni106** [comment]
> @donni106 do you think Image should be smaller ?

i think they work very well a bit wider than text. we can use it for now.
we could even decide later, if we want to have them the same width as text.

File: src/components/Wrapper.js (hunk 1 above)
Comment (SoleCincis) [reply src/components/Wrapper.js]: all right !

**Comment by SoleCincis** [comment]
small bug that I couldn't fix in detail screen with carousel images : 
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 27 
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 22

**Comment by SoleCincis** [comment]
Just in start zeit (eisenhütten) , images looks way to big 

Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 13 07 59

**Comment by SoleCincis** [comment]
> HTML screen with Image ( iPhone 11 Pro Max ) @donni106 do you think Image should be smaller ?
> Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 54 28
> Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-06 at 09 53 41

**Comment by SoleCincis** [comment]
> small bug that I couldn't fix in detail screen with carousel images :
> Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 27
> Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 22

This is happening because this branch is not inlined yet with this [one](https://github.com/ikuseiGmbH/smart-village-app-app/pull/94) where this same bug have been fixed .

File: src/screens/HtmlScreen.js [hunk 2]
```diff
@@ -45,6 +54,7 @@ export const HtmlScreen = ({ navigation }) => {
   const rootRouteName = navigation.getParam('rootRouteName', '');
   const subQuery = navigation.getParam('subQuery', '');
   const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime });
+  const { orientation } = useContext(OrientationContext);
```
Comment (donni106) [inline src/screens/HtmlScreen.js:57]: lets put this context up as well, please

**Review by SoleCincis — COMMENTED** [review]
> > small bug that I couldn't fix in detail screen with carousel images :
> > Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 27
> > Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 12 12 22
> 
> This is happening because this branch is not inlined yet with this [one](https://github.com/ikuseiGmbH/smart-village-app-app/pull/94) where this same bug have been fixed .

This merged just happened bit it seams that the problem persist in article screen : 
Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-16 at 14 03 23

File: src/screens/FormScreen.js [hunk 3]
```diff
@@ -1,20 +1,22 @@
 import PropTypes from 'prop-types';
```
[the remark this answers is not in the document]
Comment (donni106) [reply src/screens/FormScreen.js:1]: in eisenhüttenstadt. but you can put it in you local navigator to be able to navigate to from elsewhere for testing.
we can do that together.

**Comment by SoleCincis** [comment]
> Just in start zeit (eisenhütten) , image looks way to big
> 
> Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-07 at 13 07 59

This is fixed on last commit by Daniel ,now it looks good : 
Bildschirmfoto 2020-10-16 um 16 41 08

File: src/screens/FormScreen.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (SoleCincis) [reply src/screens/FormScreen.js:1]: Here it's how it looks now , well done Daniel ! 
Bildschirmfoto 2020-10-16 um 16 48 23

**Pull request merged**